src/components/commerce/CourtList.js

```diff
@@ -49,23 +84,84 @@ class CourtList extends Component {
     return <EmptyList title="No hay ninguna cancha" />;
   };
 
+  courtsFilter(text) {
+    this.setState({ search: text });
+
+    this.setState({
+      courtsSearching: this.props.courts.filter(
```

Comment:
aca me parece al pedo mandar la lista al state, porque estas como duplicando los datos al dope jaaja que directamente la funcion retorne el array, y en el faltlist, en vez de this.state.courtsSearching, le pasas la funcion

Reply:
Picanteee wasaaa